packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseHomepageV2.tsx

```diff
@@ -460,6 +460,27 @@ export function SynapseHomepageV2({ gotoPlace }: SynapseHomepageV2Props) {
               Featured datasets
             </Typography>
             <SynapseFeaturedDatasets sourceTable={featuredDatasetsTable} />
+            <Box sx={{ p: '20px 10px' }}>
```

Comment:
Add a centered button to the new "DataCatalog" GWT place

Reply:
<img width="1223" alt="image" src="https://github.com/user-attachments/assets/ef4cd5e3-81c2-4363-911b-d22641f8700a" />